ledger-api/rs-grpc-akka/src/test/scala/com/digitalasset/grpc/adapter/utils/implementations/HelloService_AkkaImplementation.scala

```diff
@@ -9,19 +9,19 @@ import akka.stream.Materializer
 import akka.stream.scaladsl.{Flow, Source}
 import com.daml.grpc.adapter.ExecutionSequencerFactory
 import com.daml.grpc.adapter.server.akka.ServerAdapter
-import com.daml.grpc.sampleservice.Responding
+import com.daml.grpc.sampleservice.HelloService_Responding
 import com.daml.platform.hello.HelloServiceGrpc.HelloService
 import com.daml.platform.hello.{HelloRequest, HelloResponse, HelloServiceGrpc}
 import io.grpc.stub.StreamObserver
 import io.grpc.{BindableService, ServerServiceDefinition}
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class AkkaImplementation(implicit
+class HelloService_AkkaImplementation(implicit
     executionSequencerFactory: ExecutionSequencerFactory,
     materializer: Materializer,
 ) extends HelloService
-    with Responding
+    with HelloService_Responding
```

Comment:
Can you please use Scala naming conventions?

Reply:
Ok